updater/spec/dependabot/updater_spec.rb

```diff
@@ -1512,18 +1970,30 @@ def expect_update_checker_with_ignored_versions(versions)
         let(:error) { Dependabot::DependencyFileNotEvaluatable.new("message") }
 
         it "doesn't tell Sentry" do
+          job = build_job
+          service = build_service(job: job)
+          updater = build_updater(service: service, job: job)
+
           expect(Raven).to_not receive(:capture_exception)
+
           updater.run
         end
 
         it "tells the main backend" do
+          job = build_job
+          service = build_service(job: job)
+          updater = build_updater(service: service, job: job)
+
+          # allow(updater).to receive(:dependency_files).and_raise(error)
+
```

Comment:
I just noticed this stray comment that's also probably a leftover of the previous FIXMEs that we ended removing?